build.gradle

```diff
@@ -0,0 +1,124 @@
+plugins {
+    id "com.jfrog.bintray" version "1.7.3"
+    id "com.github.jruby-gradle.base" version "0.1.5"
+    id "java"
+    id "checkstyle"
+    id "jacoco"
+}
+import com.github.jrubygradle.JRubyExec
+repositories {
+    mavenCentral()
+    jcenter()
+    // TODO(dmikurube): Remove this after embulk-base-restclient goes to jcenter.
+    maven {
+        url "https://dl.bintray.com/embulk-base-restclient/maven"
+    }
+}
+configurations {
+    provided
+}
+
+version = "0.3.0.1"

```

Comment:
@sakama We wanted to release a "snapshot" version before PR is merged so that we can test on console-development, hence used an additional number.